basic_module/account/src/key/src/network.rs

```diff
@@ -0,0 +1,106 @@
+// Copyright 2018, 2020 Kodebox, Inc.
+// This file is part of CodeChain.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use rlp::{Decodable, DecoderError, Encodable, Rlp, RlpStream};
+use serde::de::Error;
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+use std::fmt;
+use std::str::{self, FromStr};
+
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone, Copy, Hash)]
+pub struct NetworkId([u8; 2]);
```

Comment:
@somniumism Why did you define `NetworkId` on your own?
Since `NetworkId` is a part of `ConsensusParams`, which is managed by the host, I think you shouldn't define this by yourself.

Reply:
Why do you have a separate `key` module in your crate?